.terraform-docs.yml

```diff
@@ -0,0 +1,161 @@
+formatter: "markdown"
+
+# Tested with this version
+# As Majorversion is 0 any change may break
+version: ">=0.16.0"
+
+sections:
+  hide:
+    - header
+    - providers
+
+output:
+  file: "README.md"
+  mode: inject
+
+sort:
+  enabled: true
+  by: required
+
+content: |-
+  {{- define "setDict" -}}
+    {{- $resource := list -}}
+    {{- if hasKey .Dict .Key -}}
+      {{- $resource = get .Dict .Key -}}
+    {{- else -}}
+       {{- $resource = list -}}
+    {{- end -}}
+    {{- $resource := append $resource .Resource -}}
+    {{- $_ := set .Dict .Key $resource -}}
+  {{- end -}}
+  
+  {{- $filesResources := dict -}}
+  {{- $resourceTypes := dict -}}
+  {{- range .Module.Resources -}}
+    {{- template "setDict" dict "Dict" $filesResources "Key" .Position.Filename "Resource" . -}}
+    {{- $isResource := eq "resource" (printf "%s" .GetMode) -}}
+    {{- if $isResource -}}
+      {{- template "setDict" dict "Dict" $resourceTypes "Key" (printf "%s_%s" .ProviderName .Type) "Resource" . -}}
+    {{- end -}}
+  {{- end -}}
+  {{ indent 0 "#" }} Usage
+  
+  # Google Cloud Folder Module
+
+  This module allows the creation and management of folders, including support for IAM bindings, organization policies, and hierarchical firewall rules.
+
+  ## Examples
+
+  ### Basic
+
+  This Module creates a GCP Folder
```

Comment:
Bei der Modulbeschreibung fehlt noch die referenz zu Cloud Foundation fabric.
"Original Module from [Cloud-Foundation-Fabric](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric)"

Wollen wir den Part in die Autodoku packen oder wie beim gcs Modul die Kurzbeschreibung des Moduls und diesen Part vor die Autodoku schieben?